Error on creation of a Resource with StillImageFileValue

I have a kind of `mimetype` error when I create a resource related to an uploaded image (see below) in Sipi with `/upload?token=` endpoint. (following the doc [there](https://docs.knora.org/paradox/03-apis/api-v2/editing-values.html#upload-files-to-sipi))

It works with Salsah which uses the `thumbnail` sipi endpoint. 

I suspect that `Sipi` does not return the correct `mimetype` in `knora.json` (see log below)

error:
```json
{"error":"org.knora.webapi.TriplestoreResponseException: Triplestore responded with HTTP code 500: org.eclipse.rdf4j.repository.RepositoryException: org.eclipse.rdf4j.sail.SailException: com.ontotext.trree.consistency.ConsistencyException: Consistency check empty_string_0 failed:\nhttp://rdfh.ch/0113/yvQZ9nEyRGKyUhNeDqUnDA/values/6VCA-236RtSv2I0iO03jVg http://www.knora.org/ontology/knora-base#originalMimeType \"\"^^<http://www.w3.org/2001/XMLSchema#string>\n"}
```
some logs of stack: 
```
sipi_1     | Sipi: Accepted connection from 172.21.0.3
sipi_1     | Sipi: knora.json created from: /sipi/images/tmp/C/ACPIxZPW7pv-F5B7k0LPQAa.jp2

[...]

graphdb_1  | [INFO ] 2019-03-22 11:18:12,392 [repositories/knora-test | c.o.g.s.StatementsController] POST SPARQL update request to repository
graphdb_1  | [ERROR] 2019-03-22 11:18:12,477 [repositories/knora-test | c.o.t.SailConnectionImpl] Transaction failed. Commit will be rolled back.
graphdb_1  | com.ontotext.trree.consistency.ConsistencyException: Consistency check empty_string_0 failed:
graphdb_1  | http://rdfh.ch/0113/yvQZ9nEyRGKyUhNeDqUnDA/values/6VCA-236RtSv2I0iO03jVg http://www.knora.org/ontology/knora-base#originalMimeType ""^^<http://www.w3.org/2001/XMLSchema#string>
graphdb_1  |
graphdb_1  | 	at com.ontotext.trree.consistency.ConsistencyChecker$2.check(ConsistencyChecker.java:107)

```

The image:  ![Corcelles_Seigneux_Gabriel_2](https://user-images.githubusercontent.com/15976605/54819543-80bd2780-4c9c-11e9-8946-547d6ad3278c.JPG)

--- a/webapi/src/main/scala/org/knora/webapi/messages/store/sipimessages/SipiMessages.scala
+++ b/webapi/src/main/scala/org/knora/webapi/messages/store/sipimessages/SipiMessages.scala
@@ -333,7 +333,15 @@ case class GetImageMetadataRequestV2(fileUrl: String,
 case class GetImageMetadataResponseV2(originalFilename: String,
                                       originalMimeType: String,
                                       width: Int,
-                                      height: Int)
+                                      height: Int) {
+    if (originalFilename.isEmpty) {
+        throw SipiException(s"Sipi returned an empty originalFilename")
+    }
+
+    if (originalMimeType.isEmpty) {
+        throw SipiException(s"Sipi returned an empty originalMimeType")
+    }
+}
 
 object GetImageMetadataResponseV2JsonProtocol extends SprayJsonSupport with DefaultJsonProtocol {
     implicit val getImageMetadataResponseV2Format: RootJsonFormat[GetImageMetadataResponseV2] = jsonFormat4(GetImageMetadataResponseV2)

